name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/ibflex/IBFlexStatementExtractor.java

```diff
@@ -777,4 +775,100 @@ public List<Item> getResults()
             return results;
         }
     }
+
+    @Override
+    public List<Item> extract(SecurityCache securityCache, Extractor.InputFile inputFile, List<Exception> errors)
+    {
+        try (FileInputStream in = new FileInputStream(inputFile.getFile()))
+        {
+            IBFlexStatementExtractorResult result = importActivityStatement(in);
+            errors.addAll(result.getErrors());
+            return result.getResults();
+        }
+        catch (IOException e)
+        {
+            errors.add(e);
+            return Collections.emptyList();
+        }
+    }
+
+    @Override
+    public List<Item> postProcessing(List<Item> items)
+    {
+        // Group dividend and taxes transactions together and group by date and
```

Comment:
This is nice! One thing this will do for existing users is that if they import xmls that contain already imported transactions with the prior, non-grouped behavior, then the automatic recognition of already existing transactions will fail and things will be imported twice.

Not sure if anything can or should be done about this, but it might be good to make sure we mention it in the release notes?